server/app/views/v1/volumes/_volume.json.jbuilder

```diff
@@ -5,3 +5,13 @@ json.name volume.name
 json.scope volume.scope
 json.driver volume.driver
 json.driver_opts volume.driver_opts
+json.instances volume.volume_instances.map { |v|
+  {
+    node: v.host_node.name,
+    name: v.name
+  }
+}
+
+json.services volume.services.map { |s|
```

Comment:
both ways produce same results. But I'll change this to ^ this format for consistency sake.